### ISSUE TITLE
[FR] allow publishing in Listed

### ISSUE THREAD

**Issue by fazlerabbi37** [insanum/sncli#98]
I just found this project from one of the issues in the [sn-cli](https://github.com/jonhadfield/sn-cli) project. I was looking into it to request the same feature. See [sn-cli issue 15](https://github.com/jonhadfield/sn-cli/issues/15). I just tried the `create` and `list` commands from `sncli` and it works great for me. 

However, I was looking into Standard Notes because with [Listed.to](https://listed.to), it allows people to publish anonymous blog. I am really excited about Listed.to and it's feature. If I were to use it, I would spin up a minimal VM and install a CLI client to write and publish the blog but as of now it is not possible to do it with this program as it requires sign in and extensions are not supported as well. 

It would be very helpful for a lot of privacy concern people if this feature is implemented.

Thanks for the great tool!

**Comment by swalladge** [comment]
Hi @fazlerabbi37,   Listed.to appears to be a service that integrates with https://standardnotes.org/. This notes app is a client for https://simplenote.com/, so publishing to Listed is not something we could support.

**Comment by fazlerabbi37** [comment]
> Listed.to appears to be a service that integrates with https://standardnotes.org/

that is true. Listed.to is a service that is integrates with Standard Notes and interacts with Standard Notes via extension. 

> so publishing to Listed is not something we could support.

To use this tool to publish to Listed.to anonymously, we would need 2 functions:
- using the tool without sign in so that people can use it locally without syncing
- integration of extension

What I would like to know if you are planning to support this. If not would you mind if I take a stab at it (though I am a novice and would require a lot of your guidance)?

**Comment by swalladge** [comment]
@fazlerabbi37 I'm still not sure exactly what you are asking? Are you wanting a feature on sncli that allows pushing a note to listed.to? How does this relate to being anonymous?

> using the tool without sign in so that people can use it locally without syncing

You already can with sncli - if you don't set a username or password, it will still work fine as an offline notes app.

> What I would like to know if you are planning to support this. If not would you mind if I take a stab at it (though I am a novice and would require a lot of your guidance)?

Please do. I currently don't have much time for development here. I would be happy to give guidance on a pull request. Perhaps if you write an example proof of concept it would also help explain exactly what you have in mind.

**Comment by fazlerabbi37** [comment]
> Are you wanting a feature on sncli that allows pushing a note to listed.to? How does this relate to being anonymous?

Yes, I precisely want a feature where I can push to Listed.to using this CLI tool. This relates to being anonymous because I did not know that we can use `sncli` can be used without sign in and that defeats the purpose of anonymity.
 
> You already can with sncli - if you don't set a username or password, it will still work fine as an offline notes app.

I did not know this so thanks for this!

> Please do.

I will try to do a PoC once I get a chance to get a look at the current code I guess. Also I need to understand how the extensions work in Standard Notes.

**Comment by swalladge** [comment]
> Yes, I precisely want a feature where I can push to Listed.to using this CLI tool.

Ok, cool. In that case, yes that's a feature that could be useful if it could be implemented in a way that is off by default, doesn't cause much extra maintenance, and doesn't affect existing functionality when it's turned on.

> Also I need to understand how the extensions work in Standard Notes.

Definitely. I'm not sure if Listed.to is open source, or its terms and conditions, so it may be worth finding a contact for that, or contacting the maintainers of standardnotes. Some reverse engineering may be required...

**Comment by fazlerabbi37** [comment]
> I'm not sure if Listed.to is open source

It is open source. source code for server: https://github.com/standardnotes/listed
